src/index.html

```diff
@@ -9,6 +9,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav class="nav_1024">
+        <img src="images/logo.png"
+        class="image" alt="">
```

Comment:
```suggestion
        <img 
          src="images/logo.png"
          class="image"
          alt="logo"
        >
```
Не оставляй атрибут alt пустым
Форматируй теги таким образом